src/css/base.scss

```diff
@@ -301,6 +301,7 @@
     text-align: center;
     font-size: 25px;
     text-transform: capitalize;
+    text-transform: lowercase;
```

Comment:
this line makes the previous line useless

Reply:
Oops! Didn't mean to have that last one in. Deleted